README.md

```diff
@@ -201,6 +201,12 @@ All of the settings listed in ```settings.py``` can be overridden in a
   'state' is a defined term, it may be useful to exclude the phrase 'shall
   state'. Terms associated with the constant, `ALL`, will be ignored in all
   CFR parts parsed.
+* ```INCLUDE_DEFINITIONS_IN``` - a dictionary mapping CFR part numbers to a
+  list of tuples containing (term, context) for terms that *are
+  definitely definitions*. For example, a term that isn't necessary 
+  succeeded by subparagraphs that define it rather than phraseology 
+  like "is defined as". Terms associated with the constant, `ALL`, will 
+  be included in all CFR parts parsed.
```

Comment:
Oh, it's a beauty too:

["Empirically derived, demonstrably and statistically sound, credit scoring system" is apparently the term being defined here:](https://github.com/willbarton/fr-notices/blob/master/annual/CFR-2012-title12-vol8-part1002.xml#L107)

```xml
      <P>(p) <E T="03">Empirically derived and other credit scoring systems.</E> (1) <E T="03">A credit scoring 
system</E> is a system that evaluates an applicant's creditworthiness mechanically, based on key attributes of the 
applicant and aspects of the transaction, and that determines, alone or in conjunction with an evaluation of additional 
information about the applicant, whether an applicant is deemed creditworthy. To qualify as an <E T="03">empirically 
derived, demonstrably and statistically sound, credit scoring system,</E> the system must be:</P>
```